app/controllers/api/v1/base_controller.rb

```diff
@@ -0,0 +1,17 @@
+module Api
+  module V1
```

Comment:
I'd like to have some info here in comments about using "v1" like this -- eg, is this a recommended practice, what the rationale is -- it makes intuitive sense to me but I'd like to really understand why, and would anyway like to have an explanation here so that eg someone in future can guess when it's meant to go to v2, what that will entail/what it will break, whether deliberately or not, etc)